_opcodes/op102-bgee.html

```diff
@@ -0,0 +1,23 @@
+---
+n: 102
+name: "Protection: from Spell Levels"
+param1: "Spell Level"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Gives the targeted creature(s) protection from effects (opcodes) whose 'Power' attribute is equal to 'Spell Level'.
+<br/>Valid 'Spell Level' (Power) levels range: 0 – 9.
+
+{% capture note %}
+<ul>
+	<li>Each application of this opcode only protects against its specified 'Spell Level', not "up to" it.</li>
+	<li>Each effect in a spell/item is checked separately by this opcode.</li>
+</ul>
+{% endcapture %}
+
+{% include note.html %}
```

Comment:
No need to have this one separate, both descriptions describe the same behaviour, so you can just merge them.

Reply:
What are you referring to exactly?

The BG2 one states that this opcode protects **up to** `param1`, whereas on EE games it protects just against `param1`.

Reply:
Yes, but that's wrong. The new description is correct. Eg. if you look at GoI, it uses several opcodes to do it's job.

Reply:
OK, will merge...

Reply:
Done.